content/guides/user-guides/multi-cloud/object-archival-restore/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Archival and Retrieval
+description: "User guide for Object Archival and Retrieval"
+weight: 40
+disableToc: false
+tags: ["user guide", "archival and retrieval"] 
```

Comment:
Add tag `multi-cloud` for all the multi-cloud related guides.

Reply:
`["user guide", "archival and retrieval"]` - `["user guide", "archival and retrieval", "multi-cloud"]`

Reply:
Done